revscoring/datasources/revision_oriented.py

```diff
@@ -54,8 +55,10 @@ def __init__(self, name,
 
         self.id = Datasource(name + ".id")
         "`int` : Revision ID"
-        self.timestamp = Datasource(name + ".timestamp")
-        ":class:`mwtypes.Timestamp` : Timestamp the revision was saved"
+        self.timestamp_str = Datasource(name + ".timestamp_str")
+        "`str` : Timestamp the revision was saved in ISO format"
+        self.timestamp = Timestamp(self.timestamp_str, name + ".timestamp")
+        ":class:`mwtypes.Timestamp`: Timestamp the revision was saved"
```

Comment:
We don't implement any dependencies in datasources.revision_oriented except for this one.  It seems to break the pattern.  I think we can have `timestamp` be here and `timestamp_str` exist inside of the dependencies injected by `api.Extractor`.